### PR TITLE
GDB-7401: French translations for the workbench guide blocks

### PR DESCRIPTION
**Pull request by avataar**
Translations  for common code in the workbench that were translated with Google translate - manually corrected some of the automatic translations and removed the ⚒ (work-in-progress) symbol.

- Also reused some of the existing translations for buttons instead of creating duplicate keys

### CODE REVIEW

File: src/i18n/locale-fr.json [hunk 1]
```diff
@@ -1615,73 +1615,69 @@
...
-    "guide.button.previous": "⚒Précédent",
-    "guide.button.pause": "⚒Pause",
-    "guide.button.guide-paused": "⚒Guide en pause",
-    "guide.button.guide-resume": "⚒Continuer le guide",
+    "guide.button.guide-paused": "Guide en pause",
+    "guide.button.guide-resume": "Continuer le guide",
     "guide.validate.no-next": "Impossible de passer au suivant",
-    "guide.total.progress": "⚒Étape de guidage {{n}} de {{nn}}",
-    "guide.total.progress.tooltip": "⚒Ceci indique l'étape actuelle par rapport à l'ensemble du guide et le nombre total d'étapes dans le guide.",
+    "guide.total.progress": "Étape de guidage {{n}} de {{nn}}",
```
Comment (nikolatulechki) [inline src/i18n/locale-fr.json:1621]: {{n}} de {{nn}} --> {{n}} sur {{nn}}

File: src/i18n/locale-fr.json [hunk 2]
```diff
@@ -1615,73 +1615,69 @@
...
-    "guide.step_plugin.guide-ended.content": "⚒Ce guide est terminé.",
-    "guide.step_plugin.execute-sparql-query.query-not-same.error": "⚒La requête n'est pas la même que la requête guide !"
+    "guide.step_plugin.import_rdf_file.content": "Cliquez sur le bouton <b>{{'upload.rdf.files.label'|translate}}</b> et choisissez un fichier avec le nom <b>{{resourceFile}}</b>.",
+    "guide.step_plugin.create_repository.create_repository_button.content": "Cliquez sur le bouton <b>{{'repository.create.btn'|translate}}</b>.",
+    "guide.step_plugin.create_repository.graph_db_repository.content": "Cliquez sur le bouton <b>{{'graphdb.repo'|translate}}</b>.",
+    "guide.step_plugin.create_repository.repository_id.content": "Entrez l'ID du dépôt : <b>{{repositoryId}}.</b>",
+    "guide.step_plugin.create_repository.save_button.content": "Cliquez sur le bouton <b>{{'common.create.btn'|translate}}</b>.",
+    "guide.step_plugin.create_repository.ruleset_dropdown.content": "Choisissez l'ensemble de règles : <b>{{rulesetName}}</b>.",
+    "guide.step_plugin.enable-autocomplete.content": "Cliquez sur la case à cocher qui active l'index.",
```
Comment (nikolatulechki) [inline src/i18n/locale-fr.json:1650]: Cliquez sur la case à cocher pour activer l'indexation

File: src/i18n/locale-fr.json [hunk 3]
```diff
@@ -1615,73 +1615,69 @@
...
+    "guide.step_plugin.visual-graph-properties-side-panel-close.content": "Vous pouvez fermer le panneau en cliquant sur l'icône X.",
+    "guide.step_plugin.visual-graph-properties-focus-types.title": "Propriétés du graphique visuel",
+    "guide.step_plugin.visual-graph-properties-focus-types.content": "<b>{{'types.label'|translate}}</b> affiche tous les types RDF pour <b>{{iriLabel}}</b>.",
+    "guide.step_plugin.visual-graph-properties-focus-property.title": "Propriétés du graphique visuel",
+    "guide.step_plugin.visual-graph-properties-focus-property.content": "Les valeurs des propriétés génériques telles que <b>{{focusProperty}}</b> sont répertoriées dans des sections dédiées.",
+    "guide.step_plugin.visual-graph-expand.title": "Graphe visuel : développer un nœud",
+    "guide.step_plugin.visual-graph-expand.content": "Double-cliquez sur <b>{{iriLabel}}</b> pour développer le graphique.",
+    "guide.step_plugin.guide-ended.title": "Fin du guide",
+    "guide.step_plugin.guide-ended.content": "Ce guide est terminé.",
+    "guide.step_plugin.execute-sparql-query.query-not-same.error": "La requête n'est pas la même que la requête guide !"
```
Comment (nikolatulechki) [inline src/i18n/locale-fr.json:1682]: requête guide --> requête du guide

**Review by nikolatulechki — APPROVED** [review]
Votre français est formidable, monsieur

File: src/i18n/locale-fr.json (hunk 1 above)
Comment (avataar) [reply src/i18n/locale-fr.json]: Fixed.

File: src/i18n/locale-fr.json (hunk 3 above)
Comment (avataar) [reply src/i18n/locale-fr.json]: Fixed.

File: src/i18n/locale-fr.json (hunk 2 above)
Comment (avataar) [reply src/i18n/locale-fr.json]: Fixed.

**Pull request merged**